includes/checkout/template.php

```diff
@@ -660,53 +660,59 @@ function edd_payment_mode_select() {
 */
 function edd_show_payment_icons() {
 
-	if( edd_show_gateways() && did_action( 'edd_payment_mode_top' ) ) {
+	if ( edd_show_gateways() && did_action( 'edd_payment_mode_top' ) ) {
 		return;
 	}
 
 	$payment_methods = edd_get_option( 'accepted_cards', array() );
 
-	if( empty( $payment_methods ) ) {
+	if ( empty( $payment_methods ) ) {
 		return;
 	}
 
 	echo '<div class="edd-payment-icons">';
 
-	foreach( $payment_methods as $key => $card ) {
-
-		if( edd_string_is_image_url( $key ) ) {
+	foreach ( $payment_methods as $key => $card ) {
 
+		if ( edd_string_is_image_url( $key ) ) {
 			echo '<img class="payment-icon" src="' . esc_url( $key ) . '"/>';
-
 		} else {
+			$type = 'image';
 
-			$card = strtolower( str_replace( ' ', '', $card ) );
-
-			if( has_filter( 'edd_accepted_payment_' . $card . '_image' ) ) {
-
-				$image = apply_filters( 'edd_accepted_payment_' . $card . '_image', '' );
+			$card_lowercase = strtolower( str_replace( ' ', '', $card ) );
 
+			if ( has_filter( 'edd_accepted_payment_' . $card_lowercase . '_image' ) ) {
+				$image = apply_filters( 'edd_accepted_payment_' . $card_lowercase . '_image', '' );
 			} else {
-
-				$image = edd_locate_template( 'images' . DIRECTORY_SEPARATOR . 'icons' . DIRECTORY_SEPARATOR . $card . '.png', false );
-
-				// Replaces backslashes with forward slashes for Windows systems
-				$plugin_dir  = wp_normalize_path( WP_PLUGIN_DIR );
-				$content_dir = wp_normalize_path( WP_CONTENT_DIR );
-				$image       = wp_normalize_path( $image );
-
-				$image = str_replace( $plugin_dir, WP_PLUGIN_URL, $image );
-				$image = str_replace( $content_dir, WP_CONTENT_URL, $image );
```

Comment:
Note: Unless I'm mistaken, the removed code above is no longer needed since there aren't any paths present when loading the new SVG images.

For example, the SVG markup to load the mastercard icon looks like this:

```
<svg width="50" height="32" class="payment-icon icon-mastercard" aria-labelledby="title-5b7cf8716cc9a" role="img"><title id="title-5b7cf8716cc9a">Mastercard</title> <use href="#icon-mastercard" xlink:href="#icon-mastercard"></use> </svg>
```

Reply:
Works fine in my testing